modules/kafka/src/main/java/com/devonfw/module/producer/ProducerPayload.java

```diff
@@ -0,0 +1,136 @@
+package com.devonfw.module.producer;
+
+/**
+ * @author ravicm
+ *
+ */
+public class ProducerPayload {
+
+  private Object message;
+
+  /**
+   * Topic name. If topic name is not provided then default name will be assigned.
+   */
+  private String topic;
+
+  private String key;
+
+  private Integer partition;
+
+  private Long timestamp;
+
+  /**
+   * The constructor.
+   */
+  public ProducerPayload() {
+
+    super();
+  }
+
+  /**
+   * The constructor.
+   *
+   * @param message
+   * @param topic
+   * @param key
+   * @param partition
+   * @param timestamp
+   */
+  public ProducerPayload(Object message, String topic, String key, Integer partition, Long timestamp) {
```

Comment:
Why do we set the partition number here explicitly? Normally that is done by kafka/spring-kafka